app/views/complaints/_complaints_table_detail.html.erb

```diff
@@ -0,0 +1,22 @@
+<tr>
+  <% formatted_status = status(complaint[:attributes]) %>
+  <th scope="row" role="rowheader">
+    <span class="<%= formatted_status == "New" ? "text-bold" : "" %>">
+      <%= complaint[:id] %>
+    </span>
+  </th>
+  <td 
+    data-sort-value="<%= DateTime.parse(complaint[:attributes][:creationDate]).to_i %>"
+  >
+    <%= DateTime.parse(complaint[:attributes][:creationDate]).strftime("%m/%d/%Y") %>
+  </td>
+  <td
+    data-sort-value="<%= ComplaintsHelper::STATUS_SORT_ORDER[formatted_status] %>"
+  >
+    <%= formatted_status %>
+  </td>
+  <td><%= complaint[:attributes][:grantee] %></td>
+  <td><%= truncate(complaint[:attributes][:issue]) %></td>
+  <td> TBD </td>
```

Comment:
[question] I've been wondering why there are spaces between the tags and the text, but I didn't ask because I'm shy I guess.  Now I'm wondering....why spaces when there are text, but not spaces when there are ERB tags like `<td><%= complaint... %></td>` ?

Reply:
i find the spaces more readable -- and since they are present within the erb tags, i don't feel the need to include them in  the html tags. does that make sense?

Reply:
Yep, that makes sense!  Just haven't really come across it before, so I'm glad I asked!